gammapy/estimators/excess_map.py

```diff
@@ -81,11 +68,13 @@ def correlation_radius(self, correlation_radius):
         """Sets radius"""
         self._correlation_radius = Angle(correlation_radius)
 
-    def run(self, steps="all"):
+    def run(self, dataset, steps="all"):
```

Comment:
Please move the `steps` argument to init as well then we have full consistency with what we have for our `Maker` classes..